From cafc7db374e41e3dfe56ca2c0843320266274087 Mon Sep 17 00:00:00 2001
Message-Id: <patch-v9-01.12-cafc7db374e-20221118T110058Z-avarab@gmail.com>
In-Reply-To: <cover-v9-00.12-00000000000-20221118T110058Z-avarab@gmail.com>
References: <20220809185429.20098-1-alban.gruin@gmail.com>
	<cover-v9-00.12-00000000000-20221118T110058Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Tue, 5 Apr 2022 16:49:42 +0200
Subject: [PATCH v9 01/12] merge-index doc & -h: fix padding, labels and "()"
 use
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Make the "merge-index" doc SYNOPSIS and "-h" output consistent with
one another, and small issues with it:

- Whitespace padding, per e2f4e7e8c0f (doc txt & -h consistency:
  correct padding around "[]()", 2022-10-13).

- Use "<file>" consistently, rather than using "<filename>" in the
  "-h" output, and "<file>" in the SYNOPSIS.

- The "-h" version incorrectly claimed that the filename was optional,
  but it's not.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Documentation/git-merge-index.txt | 2 +-
 builtin/merge-index.c             | 2 +-
 t/t0450/txt-help-mismatches       | 1 -
 3 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/Documentation/git-merge-index.txt b/Documentation/git-merge-index.txt
index eea56b3154e..a297105d6d8 100644
--- a/Documentation/git-merge-index.txt
+++ b/Documentation/git-merge-index.txt
@@ -9,7 +9,7 @@ git-merge-index - Run a merge for files needing merging
 SYNOPSIS
 --------
 [verse]
-'git merge-index' [-o] [-q] <merge-program> (-a | ( [--] <file>...) )
+'git merge-index' [-o] [-q] <merge-program> (-a | ([--] <file>...))
 
 DESCRIPTION
 -----------
diff --git a/builtin/merge-index.c b/builtin/merge-index.c
index 012f52bd007..1a5a64afd2a 100644
--- a/builtin/merge-index.c
+++ b/builtin/merge-index.c
@@ -80,7 +80,7 @@ int cmd_merge_index(int argc, const char **argv, const char *prefix)
 	signal(SIGCHLD, SIG_DFL);
 
 	if (argc < 3)
-		usage("git merge-index [-o] [-q] <merge-program> (-a | [--] [<filename>...])");
+		usage("git merge-index [-o] [-q] <merge-program> (-a | ([--] <file>...))");
 
 	read_cache();
 
diff --git a/t/t0450/txt-help-mismatches b/t/t0450/txt-help-mismatches
index a0777acd667..9e73c1892ae 100644
--- a/t/t0450/txt-help-mismatches
+++ b/t/t0450/txt-help-mismatches
@@ -34,7 +34,6 @@ mailsplit
 maintenance
 merge
 merge-file
-merge-index
 merge-one-file
 multi-pack-index
 name-rev
-- 
2.38.0.1511.gcdcff1f1dc2

